parijat-frontend/package.json

```diff
@@ -27,16 +28,20 @@
   "devDependencies": {
     "babel-polyfill": "^6.26.0",
     "babel-register": "^6.26.0",
+    "cucumber": "^5.1.0",
     "eslint": "^4.5.0",
     "eslint-config-synacor": "^1.1.0",
     "expect": "^23.1.0",
     "if-env": "^1.0.0",
     "mocha": "^5.2.0",
+    "nightwatch": "^1.2.4",
+    "nightwatch-api": "^2.3.0",
     "preact-cli": "^2.0.0",
     "superstatic": "^5.0.1"
   },
   "dependencies": {
     "babel-preset-env": "^1.7.0",
+    "cucumber-pretty": "^1.5.2",
```

Comment:
Why is this added to the main dependencies section?
Can/should it be in `devDependencies`?